packages/backend/src/models/SearchModel/index.ts

```diff
@@ -126,9 +73,18 @@ export class SearchModel {
             .where('projects.project_uuid', projectUuid)
             .orderBy(searchRankColumnName, 'desc');
 
-        const subquery = SearchModel.filterByCreatedAt(
-            SpaceTableName,
-            baseSubquery,
+        filterByCreatedAt(SpaceTableName, subquery, filters);
+        filterByCreatedByUuid(
+            subquery,
+            {
+                join: {
+                    joinTableName: UserTableName,
+                    joinTableIdColumnName: 'user_id',
+                    joinTableUserUuidColumnName: 'user_uuid',
+                    tableIdColumnName: 'created_by_user_id',
+                },
+                tableName: SpaceTableName,
+            },
             filters,
         );
```

Comment:
Is it possible to return the subquery without mutating it within the function? This might be clearer.

Reply:
done here: https://github.com/lightdash/lightdash/pull/9090/commits/802ef692bbadb0da366bdc171847c204c0050cc1